### PR TITLE
Events Updated

### PR DESCRIPTION
**Pull request by sunkanmii**
This PR updates the events for the DSC website.

### CODE REVIEW

**Comment by ashinzekene** [comment]
Nice

File: index.html [hunk 1]
```diff
@@ -326,15 +326,15 @@ <h2 class="section-title">Events & Workshops</h2>
                 <div class="row">
                     <div class="col-md-4 col-12">
                         <div class="card event-card">
-                            <img class="card-img-top" data-src="images/assets/events/fbw.png" alt="Event one poster">
+                            <img class="card-img-top" data-src="images\assets\events\Open-hack-week-image.png" alt="Open Hack Week event.">
```
Comment (ashinzekene) [inline index.html]: It should be `/` instead of '\'

Comment (sunkanmii) [reply index.html]: Okay

**Comment by ashinzekene** [comment]
Nice

**Pull request merged**